polling/alternative.go

```diff
@@ -41,7 +41,7 @@ func (d *AlternativeMeDataSource) ApiUrl() string {
 
 func (d *AlternativeMeDataSource) SupportedPegs() []string {
 	// Does not have all the currencies, commodities, or crypto
-	return common.MergeLists(common.CryptoAssets, []string{"EOS", "LINK", "BAT"})
+	return common.MergeLists(common.CryptoAssets, []string{"EOS", "LINK", "BAT", "XTZ"})
```

Comment:
Tezos is related to default conversion limits in this PR?
If you add Tezos in alternative, you should update test file as well. @sigrlami

Reply:
@StarNeit that's not mine corrections, these coming from staging